packages/xo-vmdk-to-vhd/src/vmdk-read.js

```diff
@@ -306,25 +305,57 @@ export class VMDKDirectParser {
   }
 }
 
+/**
+ *
+ * the grain table is the array of LBAs (in byte, not in sector) ordered by their position in the VDMK file
+ * THIS CODE RUNS ON THE BROWSER
+ */
 export async function readVmdkGrainTable (fileAccessor) {
-  let headerBuffer = await fileAccessor(0, 512)
-  let grainAddrBuffer = headerBuffer.slice(56, 56 + 8)
+  const getLongLong = (buffer, offset, name) => {
```

Comment:
This function has not dependencies on `readVmdkGrainTable` context, you should move it out.